### ISSUE TITLE
[Issue]: Freezes (with no crash report or log info) when around chisels and bits structures

### ISSUE THREAD

**Issue by jadenquinn** [The-Nexus-Project/Limitless-7#128]
### What happened?

I was building a really stupid crafting table bit platform in the air above my base on the server y'all have, and I kept freezing, but not crashing (basically the same except one is way more cryptic), and I thought this was because I was actively building with complex modded structures on a laggy server, but I JUST crashed when I wasn't breaking or placing any bits, just walking around near them, so I don't know anymore. All I know is that it probably has something to do with Chisels and Bits or the server itself. Either way, I had to force quit and reopen my game ~6 times because of this.

Freezing, as in the game NEVER comes back and it NEVER finishes "crashing". I gave it about 5 minutes before I called it and force quit it the first time.

My system: Intel Core i7-12700k, 32GB 3600Mhz SHEEP, RTX 2070 SUPER

### Pack Version

Version: 1.14.0

### To Reproduce

I have no idea. I don't think there are any other than to just play with Chisels and Bits stuff on the official server for the pack. I was able to replicate this on a MacBook Pro 16" as well.

### Are you using Oculus?

- [ ] Yes. I am using Oculus.
- [X] No. Unmodified instance.

### Are you on a Server?

- [X] Yes.
- [ ] No.

### Relevant log output / crash report.

```shell
There is nothing in the log of note... AT ALL. Nothing that matches up with the time that the game froze either, I checked.
```

### Anything else to add?

The game never actually finishes "crashing", so there is almost nothing to go on here. I was recording for the entire duration of this happening 5 times, just not the last time since everyone had gotten off I was recording with except for me.

**Comment by jadenquinn** [comment]
It happened again.

**Comment by Gbergz** [comment]
Of course, but looking at Github is not always done 24/7 and I am a solo developer, and main focus is on Limitless 8 now.

And your issue is kind of vague I don’t know anything about it and therefore haven’t responded.

**Comment by Gbergz** [comment]
Your guess on the issue is as good as mine. So not sure .

**Comment by jadenquinn** [comment]
> Of course, but looking at Github is not always done 24/7 and I am a solo developer, and main focus is on Limitless 8 now.
> 
> And your issue is kind of vague I don’t know anything about it and therefore haven’t responded.

Maybe contact the dev of Chisels?
